site/assets/scss/_component-examples.scss

```diff
@@ -302,7 +302,7 @@
 //
 
 .highlight {
-  padding: 1rem;
+  padding: var(--bs-gutter-x);
```

Comment:
Are you sure this is related? And some of the other changes in the CSS.

Reply:
Yeah, these and other changes need another pass, but they're intentional. Gives the content a little more breathing room. Problem is I need to be careful about where I use `var()` vs fixed values because our examples would accidentally inherit those changed values.